helm/happa/templates/happaapi-ingress.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: happaapi
+  namespace: default
+  labels:
+    app: happa
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+
+    nginx.ingress.kubernetes.io/cors-allow-origin: {{ .Installation.V1.GiantSwarm.Happa.Address | quote }}
```

Comment:
Don't forget to add this property [here](https://github.com/giantswarm/happa/blob/1b76b4363c6e8e7c98448da41dabcf297d40f3f6/helm/happa/ci/default-values.yaml#L6) to make CI pass